documentation/docs/13-cli.md

```diff
@@ -27,7 +27,7 @@ Builds a production version of your app, and runs your adapter if you have one s
 
 ### svelte-kit preview
 
-After you've built your app with `svelte-kit build`, you can start the production version (irrespective of any adapter that has been applied) locally with `svelte-kit preview`. This is intended for testing the production build locally, **not for serving your app**, for which you should always use an adapter.
+After you've built your app with `svelte-kit build`, you can start the production version (irrespective of any adapter that has been applied) locally with `svelte-kit preview`. This is intended for testing the production build locally, **not for serving your app**, for which you should always use an adapter. The way your app will be served is dependent on the adapter used, see the documentation of your chosen adapter for specific instructions on how to run your app.
```

Comment:
the new sentence seems out of place to me because running your app in production with the adapter is unrelated to `svelte-kit preview`. it seems like it might belong in the `svelte-kit build` section or something instead

Reply:
I placed it here because the preceding sentence mentions it is **not for serving your app** leaving several users wonder what the correct command is for serving the app.
Ideally a comment like this (I can have another look later) would be somehow integrated in the sections about `adapters` themselves.

I feel the documentation is lacking here as this is a question that comes back every other day on the Discord in some form or another, but clearly some more discussion can be had here.

I would propose to move the wording indeed to `svelte-kit build` with something along the lines of

> After building you can run your app, see the chosen adapter for specific instructions on how to do this.

In preview remove the new lines, but add a link to the adapter section.
And in the adapter section another note that you should check the documentation of the adapter for serving instructions.

Reply:
I think the following line in the `preview` docs still needs to be more clear:

```diff
-This is intended for testing the production build locally, **not for serving your app**, , for which you should always use an adapter
+This is intended for testing the production build locally, **not for serving your app in the adapter target environment locally** (for locally testing a prod build with the adapter output see the adapter docs).
```

Or something of the sort. The distinctions I see that need to be made are:

1. dev: `svelte-kit dev`
2. build prod (used for all subsequent steps): `svelte-kit build`
3. test prod build locally without adapter environment: `svelte-kit preview`
4. test prod build locally with adapter environment: see adapter docs
5. deploy prod build to adapter environment: see adapter docs

with steps 3 and 4 being the confusing pieces. 3 relates to `preview` so should be there. 4 relates to `build` so should be there.

The docs don't make these 5 steps clear in a concise manner. You have to read a lot of documentation to gather the information on these 5 steps.